public/js/teacher.js

```diff
@@ -0,0 +1,13 @@
+(function() {
```

Comment:
Yea in the future there will be an ajax request so I just wanted to make two separate files so that I can change it easier in the future.